packages/ra-ui-materialui/src/button/ExportButton.tsx

```diff
@@ -13,6 +13,7 @@ import {
     useResourceContext,
 } from 'ra-core';
 import { Button, ButtonProps } from './Button';
+import { UseQueryOptions } from 'react-query';
```

Comment:
linter warning on this line:
`'UseQueryOptions' is defined but never used`

Reply:
cleaned up